### PR TITLE
docfix: Updated power daemon service name for Fedora

### PR DESCRIPTION
**Pull request by TopherIsSwell**
The szydell/system76 COPR repo for fedora provides the system76-power
    package and as of the latest version of it
    (system76-power-1.1.23-1.x86_64), the system76-power unit is named
    `com.system76.PowerDaemon.service`. This patch updates this name in the
    locations where its referenced in the documentation.

    ```bash
    $ rpm -ql system76-power
    /etc/modprobe.d/system76-power.conf
    /etc/modules-load.d/system76-power.conf
    /usr/bin/system76-power
    /usr/lib/.build-id
    /usr/lib/.build-id/f8
    /usr/lib/.build-id/f8/e399a6d38e428f12d3b921b6aef806e3ccda0e
    /usr/lib/systemd/system/com.system76.PowerDaemon.service
    /usr/lib/systemd/system/system76-power-wake.service
    /usr/share/bash-completion/completions/system76-power
    /usr/share/dbus-1/interfaces/com.system76.PowerDaemon.xml
    /usr/share/dbus-1/system.d/com.system76.PowerDaemon.conf
    /usr/share/polkit-1/actions/com.system76.PowerDaemon.policy
    ```

### CODE REVIEW

**Review by n3m0-22 — APPROVED** [review]
I can confirm this fixes the issues with the current documentation when setting up Fedora. Regarding the final command changed `sudo systemctl status system76-power.service`. This command still works, but I do think for the sake of consistency changing it to `sudo systemctl status com.system76.PowerDaemon.service` makes sense.

File: content/system76-software.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ Now log out and use the <u>Extensions</u> to enable the extenstion.
 If `Gio.DBusError` presents itself when attempting to enable the extension within the Extensions application, confirm that the `system76-power.service` is running:
```
Comment (jacobgkau) [inline content/system76-software.md:199]: The old/alias name is still being used on this line.

Comment (jacobgkau) [inline content/system76-software.md:199]: The old/alias name is still being used on this line.

Comment (n3m0-22) [reply content/system76-software.md]: Nice catch. I believe https://github.com/system76/docs/pull/1073/commits/8f7f9f1143403ead1cfb6e11a840698a04976086 resolves the last two instances of the old name.

**Pull request merged**